src/index.html

```diff
@@ -1,13 +1,395 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" type="image" href="images/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#menu" class="icon icon--menu"></a>
+            <a href="#" class="logo">
+              <img
+              class="logo__image"
+              src="images/logo.png"
+              alt="Miami Condo Kings"
+              >
```

Comment:
Pay attention to style code

```suggestion
              <img
                class="logo__image"
                src="images/logo.png"
                alt="Miami Condo Kings"
              >
```